wasmCloud/wasmCloud
# ci: rework

- Update https://github.com/rvolosatovs/nixify, which now has support for cross-compilation to win64 and static armv7 targets (disabled for now, win64 will be done in #295 )
- Configure Nix caches within the flake

## Related Issues


Partially extracted from #293

## Release Information


## Consumer Impact


## Testing



Built on platform(s)


Tested on platform(s)

### Unit Test(s)


### Acceptance or Integration


### Manual Verification


- `nix flake check -L` on `x86_64-linux`
- `nix build -L .\#packages.aarch64-darwin.wasmcloud-x86_64-unknown-linux-musl` (remote build on `aarch64-darwin` from `x86_64-linux`)
- `nix build -L .\#packages.x86_64-darwin.wasmcloud-x86_64-unknown-linux-musl` (remote build on `x86_64-darwin` from `x86_64-linux`)
- `nix build -L .\#wasmcloud-aarch64-unknown-linux-musl` (remote build on `x86_64-linux` from `x86_64-linux`)
- etc...
